packages/accordion-react/src/AccordionItem.tsx

```diff
@@ -6,7 +6,7 @@ import React, { ReactNode, useState } from "react";
 import { useAnimatedHeight } from "@fremtind/jkl-react-hooks";
 
 interface Props {
-    title: string;
+    title: string | ReactNode;
```

Comment:
No need for both, since `string` is a part of the `ReactNode` type. However it might be worth keeping it just for clarity?

Reply:
Agree that it is nice to keep for clarity